modules/admin_manual/pages/configuration/user/oidc/ms-azure-setup.adoc

```diff
@@ -143,8 +143,8 @@ If you want to let ownCloud create users which are not present during a OIDC aut
     ],
 ----
 
-////
-=== Register ownCloud Clients
+=== Register ownCloud Desktop and Mobile Clients
 
 To allow the ownCloud clients (Web/desktop/Android/iOS) to interact with the identity provider, you have to register them as clients.
-////
+
+Only branded ownCloud desktop and mobile clients can be registered with Azure AD, because they need to be build with customized `scope` and `prompt` parameters. ownCloud full branding subscription is needed. Please get in touch with ownCloud Consulting for more help.
```

Comment:
> Please get in touch with ownCloud Consulting for more help

Should it be `support` instead of consulting? This would be the first mentioning of consulting as far I can remember.
In case, I could make a suggestable commit using links we use when mentioning support.

Reply:
I saw consulting was a used a few times: https://www.google.com/search?q=site%3Adoc.owncloud.com%2Fserver+consulting

@JRundfeldt help with Azure AD + branded clients: Support or Consulting?

Reply:
Just fyi, we have links (via our homepage) to support but not to consulting. Asking to guide customers where to go...

Reply:
"Support" is more adequate, as they are the central ingresspoint for nearly all customer contacts. "Sales" or "your account manager, if you already have an enterprise subscription" would be even more fitting.

Reply:
Thanks, will rephrase the sentence accordingly.